core/src/com/unciv/models/ruleset/Victory.kt

```diff
@@ -223,10 +227,13 @@ class Milestone(val uniqueDescription: String, private val parentVictory: Victor
                 val majorCivs = civInfo.gameInfo.civilizations.filter { it.isMajorCiv() }
                 val civReligion = civInfo.religionManager.religion
                 for (civ in majorCivs) {
+                    val milestoneText =
+                        if (civInfo.knows(civ) || !civ.isAlive()) "Majority religion of [${civ.civName}]"
+                        else "Majority religion of [${Constants.unknownNationName}]"
                     if (civReligion == null || (civReligion.isPantheon() && civInfo != civ) || !civ.religionManager.isMajorityReligionForCiv(civReligion))
-                        buttons.add(getMilestoneButton("Majority religion of [${civ.civName}]", false))
+                        buttons.add(getMilestoneButton(milestoneText, false))
                     else
-                        buttons.add(getMilestoneButton("Majority religion of [${civ.civName}]", true))
+                        buttons.add(getMilestoneButton(milestoneText, true))
```

Comment:
Send to me the second parameter passed to this function is the reverse if the if, so we can replace the if with a parameter

Reply:
Those isn't a problem with your code though, existed before, so I approve :)

Reply:
Done, thanks to De Morgan's laws.